xgcm/test/test_metric_ops_multi_axis.py

```diff
@@ -5,14 +5,18 @@
 from xgcm.test.datasets import datasets_grid_metric
 
 
-def _expected_result(da, metric, grid, dim, axes, funcname, boundary=None):
+def _expected_result(
+    da, metric, grid, dim, axes, funcname, boundary=None, fill_value=None
```

Comment:
This does not actually test any of the boundary conditions? I would parametrize over some options (at least for `boundary`).

Reply:
because the function `_expected_result` is used inside other functions:
```python
def test_bgrid(self, funcname, boundary, periodic, fill_value):
        ds, coords, metrics = datasets_grid_metric("B")
        grid = Grid(ds, coords=coords, metrics=metrics, periodic=periodic)

        func = getattr(grid, funcname)
        kwargs = dict(boundary=boundary, fill_value=fill_value)

        # test tracer position
        for axis, metric_name, dim in zip(
            ["X", "Y", "Z", ["X", "Y"], ["X", "Y", "Z"]],
            ["dx_t", "dy_t", "dz_t", "area_t", "volume_t"],
            ["xt", "yt", "zt", ["xt", "yt"], ["xt", "yt", "zt"]],
        ):
            new = func(ds.tracer, axis, **kwargs)
            expected = _expected_result(
                ds.tracer,
                ds[metric_name],
                grid,
                dim,
                axis,
                funcname,
                boundary,
                fill_value,
            )
            assert_allclose(new, expected)
```
the `boundary` value will differ depending on the parameters used per iteration, and `boundary` is parameterized fro `test_bgrid`.